wg-ai-conformance/charter.md

```diff
@@ -0,0 +1,43 @@
+# WG Kubernetes AI Conformance Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The goal of this group is to define a standardized set of capabilities, APIs, and configurations that a Kubernetes cluster must offer to reliably and efficiently run AI/ML workloads. This initiative aims to simplify AI/ML operations on Kubernetes, accelerate adoption, guarantee interoperability and portability for AI workloads, and enable ecosystem growth on an industry-standard foundation.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- The primary artifact will be the Kubernetes AI Conformance specification and a suite of tests to demonstrate conformance.
```

Comment:
@fedebongio I guess, we should change this, right ?

```suggestion
- The primary artifact will be the CNCF Kubernetes AI Conformance specification and a suite of tests to demonstrate conformance.
```

Reply:
(that can happen later)